### PR TITLE
ReadMe - Product Description

### PR DESCRIPTION
**Pull request by yash-chowdhary**
Issue #49

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -15,14 +15,17 @@ ifndef::env-github[]
...
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
-* It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
-* It is *written in OOP fashion*. It provides a *reasonably well-written* code example that is *significantly bigger* (around 6 KLoC)than what students usually write in beginner-level SE modules.
-* What's different from https://github.com/se-edu/addressbook-level3[level 3]:
-** A more sophisticated GUI that includes a list  panel and an in-built Browser.
-** More test cases, including automated GUI testing.
-** Support for _Build Automation_ using Gradle and for _Continuous Integration_ using Travis CI.
-
+* EXCOntact is a desktop application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
+* EXCOntact is targeted to Student Organizations like clubs and societies characterized by a well-established hierarchy.
```
Comment (amrut-prabhu) [inline README.adoc]: targeted at*
(I'll fix it when I add other stuff later)

File: README.adoc [hunk 2]
```diff
@@ -15,14 +15,17 @@ ifndef::env-github[]
...
+* Student Organizations require a one-stop destination for all their managerial needs when it comes to enrollment, delegating tasks, preparing for events, and opening polls and feedback systems.
+* Student organizations can have hundreds of members which are further divided into sub-committees that have narrower focuses. These subcommittee members are often assigned individual tasks. With time, it will get very difficult to keep track of everything.
+* EXCOntact is the solution to this problem as it aims to help organizations organize better.
+* EXCOntact has the following features:
+** EXCO Members can manage the member roster
+** EXCO Members can delegate tasks to Sub-Committee Members
+** EXCO Members can set up polls and feedback surveys
+** Members can send e-mails to any other members
+** Members can keep track of upcoming events and deadlines for tasks
 == Site Map
```
Comment (amrut-prabhu) [inline README.adoc]: Need an empty line before this? o_O

Comment (yash-chowdhary) [reply README.adoc]: fixed

File: README.adoc [hunk 3]
```diff
@@ -15,13 +15,17 @@ ifndef::env-github[]
...
 endif::[]
 
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
-* It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
-* It is *written in OOP fashion*. It provides a *reasonably well-written* code example that is *significantly bigger* (around 6 KLoC)than what students usually write in beginner-level SE modules.
-* What's different from https://github.com/se-edu/addressbook-level3[level 3]:
-** A more sophisticated GUI that includes a list  panel and an in-built Browser.
-** More test cases, including automated GUI testing.
-** Support for _Build Automation_ using Gradle and for _Continuous Integration_ using Travis CI.
+* EXCOntact is a desktop application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
```
Comment (amrut-prabhu) [inline README.adoc]: Add GUI to glossary and link

**Pull request merged**